src/org/folio/Constants.groovy

```diff
@@ -133,6 +133,7 @@ class Constants {
   static String ECR_FOLIO_REPOSITORY = '732722833398.dkr.ecr.us-west-2.amazonaws.com'
   static String ECR_FOLIO_REPOSITORY_CREDENTIALS_ID = 'aws-ecr-rw-credentials'
   static String DOCKER_K8S_CLIENT_IMAGE = 'alpine/k8s:1.22.15'
+  static String FOLIO_CI_REPOSITORY = 'folioci'
```

Comment:
In my humble opinion, this constant is not necessary for now.

Reply:
Not used Constant is removed.